bin/fury

```diff
@@ -7,4 +7,8 @@ require 'rubygems'
 require 'gemfury'
 require 'gemfury/command'
 
+unless ENV['FURY_ENABLE_WARNINGS'] == '1'
+  $VERBOSE=nil
+end
```

Comment:
```suggestion
if defined?(Warning)
  Warning[:deprecated] = !!ENV['DEBUG']
end
```

Reply:
@rykov it's only in 2.7 that made this a major case. in previous versions, at least those from our test suites, they don't have the warnings. we can go with this assumption.

Wasn't aware about the 'Warning' constant, will review, and use this approach, instead of $VERBOSE. I used that, as historically it's what's commonly used to prevent warning messages. No other reason for it.